app/src/main/java/com/jmanday/sparky/MainActivity.kt

```diff
@@ -26,6 +32,10 @@ class MainActivity : ComponentActivity() {
                 }
             }
         }
+
+        val repository = DefaultAuthRepository(RemoteDataSource())
+        val viewModel = AuthViewModel(repository)
+        viewModel.login("jmanday@gmail.com", "12a34b56D1")
```

Comment:
I'm aware this is just for testing, but nevertheless I wouldn't commit such code. Everything you create a PR for is an intend to merge this into a "real" work branch, so this should not contain any test code or commented out code. It should be code you want to have like that on the main/develop branch

Reply:
Sure! They will be removed. As you say we never should do that.